docs/userguide.md

```diff
@@ -86,13 +86,16 @@ federation-v2 control plane.
 
 ## Automated Deployment
 
-If you would like to have the deployment of the federation-v2 control plane
-automated, then invoke the deployment script by running:
+Assume you are runnig the autoamted deloyment in cluster1. If you would like
+to have the deployment of the federation-v2 control plane automated, then invoke
+the deployment script by running:
 
 ```bash
 ./scripts/deploy-federation-latest.sh cluster2
 ```
 
+The above command will add both cluster1 and cluster2 to the federation control panel.
```

Comment:
I would mention "join" instead of "add" just to match the `kubefed2` command.

Reply:
Done